angular/src/lib/input/examples/default.component.ts

```diff
@@ -1,60 +1,351 @@
-import { Component } from '@angular/core';
-import { FormControl, FormGroup, FormBuilder } from '@angular/forms';
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { NgForm, FormBuilder, NgModel } from '@angular/forms';
 import { Validators, AbstractControl } from '@angular/forms';
 
 @Component({
   selector: 'example-input-default',
   template: `
-    <form [formGroup]="inputForm">
-      <md-input
-        formControlName="inputControl"
-        inputSize="small-5"
-        label="Default Input"
-        htmlId="example-default-input"
+    <md-input-group
+      label='Success not Reactive'
+      containerSize='small-12'
+      [messageArr]="[{
+        message: 'This is where the success message would be.
+          This is where the success message would be.
+          This is where the success message would be.
+          This is where the success message would be.
+          This is where the success message would be.
+          This is where the success message would be.',
```

Comment:
this seems a bit repetitive

Reply:
The point is a long string to make sure we capture the ellipsis in the kitchen sink.

Reply:
lol